### DISCUSSION TITLE
table如何实现有滚动条的时候滚动条一直出现滚动条，而不是滚动的时候出现滚动条

### DISCUSSION THREAD

**Discussion by congshu-du** [ant-design/ant-design#44856] [Q&A]
因为有些用户虽然使用的mac,但是使用鼠标，所以他没有办法点到滚动条，所以无法滚动。

**Comment by yoyo837** [comment]
这个无法控制一直出现吧, mac设置为滚动时出现, 我理解就是系统级别的设置. 此时用户滚轮无法滚动吗?

**Reply by yoyo837** [reply]
<img width="478" alt="image" src="https://github.com/ant-design/ant-design/assets/6134547/66289d1d-5a46-44c9-8c1e-2bbc45b75498">

**Reply by congshu-du** [reply]
有的人也不会设置，客户也不可能一个个去说，在代码里面可以通过修改什么样式来实现吗？

**Reply by yoyo837** [reply]
那是系统级别的实现. js 控制不了

**Reply by yoyo837** [reply]
用mac不知道这个, 是首次使用吧, 碰壁一次(学习一次)就自然知道了.

**Reply by yoyo837** [reply]
首次进入页面加提示啊, 可以用https://ant.design/components/tour-cn

**Reply by congshu-du** [reply]
有好多年龄大的用户，确实不会，遇到的太多了，我们之前都是通过重写滚动条来实现，但是自己写的滚动条优化的一直不太好。

**Reply by congshu-du** [reply]
好吧，非常感谢，我们再讨论讨论。

**Reply by liulei-studyshx** [reply]
您好，一直显示滚动条的问题解决了吗
